cirq/ops/fsim_gate.py

```diff
@@ -56,6 +56,13 @@ class FSimGate(gate_features.TwoQubitGate,
     """
 
     def __init__(self, theta: float, phi: float):
+        """
+        Args:
+            theta: Rotation angle, in radians, for the exchange interaction. Controls the inner 2x2 block of the
+                operation's unitary (the middle 2x2 block will equal `cirq.unitary(cirq.Rx(theta))`).
```

Comment:
What does this add that is not in the definition of the class?

The temrs "middle block" and "bottom 1x1 cell" aren't really clear terms in my opinion.

Reply:
It provides useful pop-up documentation when creating the class in pycharm. It places the definition directly adjacent to the use.